packetfu/packetfu
# Bump gem version 1.12

Updates version, and hey turns out I lost my old signing cert, so made a new one.

Verification steps, for people who aren't me:

- [ ] `gem build packetfu.gemspect`
- [ ] **Verify** packetfu-1.12.gem is built (WARNINGs are okay, see below)
- [ ] `gem install packetfu-1.12.gem -P HighSecurity`
- [ ] **Verify** the gem is installed without error.

This will work because you're building without my private key, and hilariously, this is never mentioned as a WARNING in either the build or in the installation. Sooooo yeah.

The other warnings should be taken care of someday, but that's what 1.13-pre will be for. Along with the other straggling PRs here.

```
todb@ubuntu:~/git/packetfu$ gem build packetfu.gemspec WARNING:  license value 'BSD' is invalid.  Use a license identifier from
http://spdx.org/licenses or 'Nonstandard' for a nonstandard license.
Did you mean '0BSD'?
WARNING:  open-ended dependency on pcaprub (>= 0) is not recommended
  if pcaprub is semantically versioned, use:
    add_runtime_dependency 'pcaprub', '~> 0'
WARNING:  open-ended dependency on rake (>= 0, development) is not recommended
  if rake is semantically versioned, use:
    add_development_dependency 'rake', '~> 0'
WARNING:  open-ended dependency on rspec (>= 0, development) is not recommended
  if rspec is semantically versioned, use:
    add_development_dependency 'rspec', '~> 0'
WARNING:  open-ended dependency on rspec-its (>= 0, development) is not recommended
  if rspec-its is semantically versioned, use:
    add_development_dependency 'rspec-its', '~> 0'
WARNING:  open-ended dependency on sdoc (>= 0, development) is not recommended
  if sdoc is semantically versioned, use:
    add_development_dependency 'sdoc', '~> 0'
WARNING:  open-ended dependency on pry (>= 0, development) is not recommended
  if pry is semantically versioned, use:
    add_development_dependency 'pry', '~> 0'
WARNING:  open-ended dependency on coveralls (>= 0, development) is not recommended
  if coveralls is semantically versioned, use:
    add_development_dependency 'coveralls', '~> 0'
WARNING:  See http://guides.rubygems.org/specification-reference/ for help
  Successfully built RubyGem
  Name: packetfu
  Version: 1.1.12
  File: packetfu-1.1.12.gem
```

I talked with @claudijd today a bit and he won't have a ton of bandwidth to test this out. I wonder if I can get @bcook-r7 to look at it and tell me if this will work for Metasploit.
